state/txn.go

```diff
@@ -67,6 +70,86 @@ func newTxn(snapshot readSnapshot) *Txn {
 	}
 }
 
+// GetDumpTree function returns accounts based on the selected criteria.
+func (txn *Txn) GetDumpTree(dumpObject *Dump, opts *DumpInfo, deleteEmptyObjects bool) ([]byte, error) {
+	if err := txn.CleanDeleteObjects(deleteEmptyObjects); err != nil {
+		return nil, err
+	}
+
+	var (
+		nextKey         []byte
+		startValid      = opts.Start == nil || len(opts.Start) == 0 || bytes.Equal(opts.Start, types.EmptyRootHash.Bytes())
```

Comment:
```suggestion
		startValid      = opts.Start == nil || len(opts.Start) == 0 || bytes.Equal(opts.Start, types.EmptyRootHash.Bytes())
```
I would suggest renaming this variable. I took me some time to figure out for what is used.
Maybe it would be more understandable with contra logic, something like this:
```suggestion
		hasStartKey      = opts.Start != nil && len(opts.Start) > 0 && !bytes.Equal(opts.Start, types.EmptyRootHash.Bytes())
```
This way start or 1st key is passed and we will search for this key until we find it and then we can reset this variable:
if hasStartKey {
    if !bytes.Equal(opts.Start, k) {
        return false
    }

    hasStartKey = false
}